keyboards/trojan_pinata/model_b/rev0/keymaps/default/keymap.c

```diff
@@ -0,0 +1,654 @@
+/* Copyright 2024 Brian H (@Trojan_Pinata)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+// screen aliases
+#define MTNS 0
+#define INFO 1
+#define LOGO 2
+#define MEW  3
+#define CEL  4
+#define RAY  5
+#define NUM_MODES 6
+uint8_t display_mode = INFO;    // defaults to 1 in case of no value
+
+// key that changes OLED
+#define OLED_MOD KC_F13
```

Comment:
I would suggest using either a higher F-keycode, or a custom keycode, as F13 is used by multiple common operating systems.